cmd/component.go

```diff
@@ -110,15 +111,26 @@ var componentDeleteCmd = &cobra.Command{
 		log.Debugf("args: %#v", strings.Join(args, " "))
 
 		componentName := args[0]
+		var confirmDeletion string
 
-		// no flag was set, create empty component
-		output, err := component.Delete(componentName)
-		if err != nil {
-			fmt.Println(err)
-			os.Exit(1)
+		if componentForceDeleteFlag {
+			confirmDeletion = "y"
+		} else {
+			fmt.Printf("Are you sure you want to delete the component: %v? [y/N] ", componentName)
```

Comment:
It should display from what application the component will be deleted.

Reply:
fixed